jailer/src/env.rs

```diff
@@ -295,20 +299,23 @@ impl Env {
                 .map_err(Error::CloseDevNullFd)?;
         }
 
-        Err(Error::Exec(
-            Command::new(chroot_exec_file)
-                .arg(format!("--id={}", self.id))
-                .arg(format!("--seccomp-level={}", self.seccomp_level))
-                .arg(format!("--start-time-us={}", self.start_time_us))
-                .arg(format!("--start-time-cpu-us={}", self.start_time_cpu_us))
-                .arg(format!("--api-sock=/{}", socket_file_name))
-                .stdin(Stdio::inherit())
-                .stdout(Stdio::inherit())
-                .stderr(Stdio::inherit())
-                .uid(self.uid())
-                .gid(self.gid())
-                .exec(),
-        ))
+        let mut command = Command::new(chroot_exec_file);
+        let init_command = command
```

Comment:
nit: do you need to use another variable (`init_command`)?

Reply:
Yes, in order to increase the lifetime of the Command object created by `new() `function.

Reply:
Wouldn't something like this work also?
```
        let mut command = Command::new(chroot_exec_file);

        command
            .arg(format!("--id={}", self.id))
            // ...
            .gid(self.gid());

        if let Some(json) = self.config_json {
            command.arg(format!("--vmm-config={}", json));
        }
        Err(Error::Exec(command.exec()))
```

Reply:
Yep, this works too, didn't try that.